static/css/mozsearch.css

```diff
@@ -1819,3 +1819,22 @@ g.true-diag g.node polygon {
 #query-debug-results-json[aria-hidden] {
   display: none;
 }
+
+/* No uses styling */
+.search-note {
+  margin: 8px 0;
+  padding-left: 10px;
+  border-left: 4px solid #6a7adb;
+  font-size: 14px;
+}
+
+.search-note strong {
+  display: block;
+  margin-bottom: 2px;
+  font-weight: 600;
+}
+
+.search-note-muted {
+  font-size: 12px;
+  color: #555;
```

Comment:
Here too, please define a variable for it, and verify the rendering both on light mode and dark mode.